sass/16-extending.md

```diff
@@ -0,0 +1,482 @@
+XHP comes with classes for all standard HTML tags, but since these are first-class objects, you can create your own XHP classes for rendering
```

Comment:
Does adding this file really belong on the rename PR?

Reply:
huh -- bizarre. Not sure why this was a new addition.